scripts/setup_kubernetes.sh

```diff
@@ -13,6 +13,6 @@ tar xf doctl-1.14.0-linux-amd64.tar.gz
 chmod +x ./doctl
 sudo mv ./doctl /usr/local/bin/doctl
 
-doctl auth init --access-token $DOCTL_ACCESS_TOKEN
+doctl auth --context develop init --access-token $DIGITALOCEAN_ACCESS_TOKEN
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-11T15:35:49Z" title="Monday, November 11th 2019, 4:35:49 pm +01:00">Nov 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -13,6 +13,6 @@ tar xf doctl-1.14.0-linux-amd64.tar.gz
 chmod +x ./doctl
 sudo mv ./doctl /usr/local/bin/doctl
 
-doctl auth init --access-token $DOCTL_ACCESS_TOKEN
+doctl auth --context develop init --access-token $DIGITALOCEAN_ACCESS_TOKEN
```


Yes, so much better! I haven't really understood the documentation of the `context` though. https://github.com/digitalocean/doctl I don't know the use case of it.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-11T18:24:46Z" title="Monday, November 11th 2019, 7:24:46 pm +01:00">Nov 11, 2019</time>_

---


it allows you to use multiple digital ocean accounts
https://github.com/digitalocean/doctl#logging-in-to-multiple-digitalocean-accounts